ClearKeep/Scenes/Social Login/Views/SocialVerify.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  SocialThanos.swift
+//  ClearKeep
+//
+//  Created by đông on 09/03/2022.
+//
+
+import SwiftUI
+import Combine
+import Common
+import CommonUI
+
+private enum Constant {
+	static let spacerTopView = 50.0
+	static let spacerBottomView = 20.0
+	static let spacer = 25.0
+	static let spacerBottom = 45.0
+	static let widthLogo = 160.0
+	static let heightLogo = 120.0
```

Comment:
Make logo is common view if need to re-use  

```suggestion
	static let logoSize = CGSize(width: 160.0, height: 120.0)
```